internal/dataplane/kong_client.go

```diff
@@ -759,6 +759,13 @@ func (c *KongClient) sendToClient(
 		AppendStubEntityWhenConfigEmpty: !client.IsKonnect() && config.InMemory,
 	}
 	targetContent := deckgen.ToDeckContent(ctx, logger, s, deckGenParams)
+	customEntities := map[string][]map[string]interface{}{}
```

Comment:
```suggestion
	customEntities := make(map[string][]map[string]any)
```
like that, it's a little easier to follow (at least for me)